demo/gallery/vat-alice.js

```diff
@@ -151,7 +154,9 @@ function makeAliceMaker(E, log) {
         async doTapFaucetAndStore() {
           log('++ alice.doTapFaucetAndStore starting');
           const pixelPaymentP = E(gallery).tapFaucet();
-          const pixelIssuer = E(pixelPaymentP).getIssuer();
+          const { pixelIssuer, useRightIssuer, transferRightIssuer } = await E(
+            gallery,
```

Comment:
Can there ever be more than one argument to E()?

Reply:
That is a good question. I think there can only be one argument. In this case, ES6 allows us to immediately destructure the object we receive so from E's perspective, it's one argument still.

Reply:
No, but prettier may well be "forcing" this extra comma because it doesn't know this, and we usually want the extra comma when a function call is laid out like this.

Reply:
Ah, whoops, didn't see the last line there in which the comma occurs. This comment makes more sense now